content/en/blog/_posts/2024-04-23-recursive-read-only-mounts.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blog
+title: 'Kubernetes 1.30: Read-only volume mounts can be finally literally read-only'
+date: 2024-04-23
+slug: recursive-read-only-mounts
+---
+
+**Author:** Akihiro Suda (NTT)
+
+Kubernetes 1.30 added alpha support for "recursive read-only mounts".
+
+Prior to this release, read-only volume mounts were surprisingly not really read-only under specific conditions.
+
+### Read-only volume mounts are not really read-only
```

Comment:
```suggestion
### Read-only volume mounts were not really read-only
```
?

Reply:
Bear in mind that people will be reading this after the v1.30 release has happened.

Reply:
Strictly speaking, volumes which are read-only (and not explicitly RRO) *are* still exposed this way.

Reply:
What about "Read-only volume mounts are not really read-only by default"

Reply:
Reworded as "Read-only volume mounts are not really read-only by default"